src/screens/GroupInfo/elements/ItemLink.js

```diff
@@ -40,13 +40,13 @@ const styles = StyleSheet.create({
     fontFamily: fonts.inter[600],
     fontSize: 14,
     lineHeight: 16.94,
-    color: COLORS.white
+    color: COLORS.almostBlack
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The text color has been changed from `COLORS.white` to `COLORS.almostBlack`. This should improve readability if the background is light, but make sure it doesn't clash with any surrounding elements.
```diff
-    color: COLORS.white
+    color: COLORS.almostBlack
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->